test/claim.test.ts

```diff
@@ -33,10 +33,6 @@ describe('StakingPool.claim', () => {
     return await setTestEnv();
   }
 
-  after(async () => {
```

Comment:
I wanted to figure out what this hook was doing because I referred it somewhere without consideration. but now it doesn't matter whatever it was 😃

Reply:
Thanks! If it seems necessary, I will comment.